test/builders/blocks.go

```diff
@@ -76,6 +80,30 @@ func (b *blockPair) WithHeight(blockHeight primitives.BlockHeight) *blockPair {
 	return b
 }
 
+func (b *blockPair) WithPrevBlockHash(prevBlock *protocol.BlockPairContainer) *blockPair {
```

Comment:
I suggest we differentiate lean helix block pair from benchmark block pair

Reply:
done, we now have 3 methods:

`LeanHelixBlockPair`
`BenchmarkConsensusBlockPair`
`BlockPair` - if you don't care which one it directs to `LeanHelixBlockPair`.. we can kill this one but then we break lots of code in the system